new_platforms/vsextension/ProjectTemplates/oeenclave/linux_gcc.mak

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+OE_SDK_ROOT_PATH = ../../../../../
```

Comment:
How does this work if a developer creates projects outside the Open Enclave source tree? Also, none of the `*.mk` files are in the Nuget package.